### PR TITLE
Refactored for better routing for use with proxy server. Added Cors.

### PR DESCRIPTION
**Pull request by theofficetroll**
https://trello.com/c/FtNI2LJa

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -45,7 +47,6 @@ app.get('/tiersIncluded/:bundleId', (req, res) => {
     }
   })
   .then(data => {
-    console.log('tier 1', data[0].dataValues.tierId);
     let tiersData = {
       tier1Id: data[0].dataValues.tierId,
       tier2Id: data[1].dataValues.tierId,
```
Comment (redperezz) [inline server/index.js:52]: ✓ edited out console log and added cors, got it

File: src/index.jsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ class Tiers extends React.Component {
     var bundleId = url.searchParams.get('bundle');
     $.ajax({
       method: 'GET',
-      url: '/tiersIncluded/' + bundleId,
+      url: 'http://localhost:3101/tiersIncluded/' + bundleId,
       datatype: 'json',
       success: (data) => {
         this.setState({
```
Comment (redperezz) [inline src/index.jsx:31]: ✓

File: src/components/tier.jsx [hunk 3]
```diff
@@ -42,7 +41,6 @@ class Tier extends React.Component {
             item2: data.item2,
```
Comment (redperezz) [inline src/components/tier.jsx:41]: I'm probably not seeing the full picture here inside the for loop so correct me if I'm wrong but wouldn't every loop through data.length just keep setting the state of item1 & item2 to data.item1 & data.item2? If you're using for testing purposes ignore this comment lol

**Review by redperezz — APPROVED** [review]
Nice work so far!

File: src/components/tier.jsx (hunk 3 above)
Comment (theofficetroll) [reply src/components/tier.jsx:41]: Yeah, my next PR fixed this, I believe. Good catch :)

**Pull request merged**